Storage/Rust/RustPlaces.swift

```diff
@@ -22,10 +22,13 @@ public class RustPlaces {
     public fileprivate(set) var isOpen: Bool = false
 
     private var didAttemptToMoveToBackup = false
+    private var notificationCenter: NotificationCenter
 
-    public init(databasePath: String) {
+    public init(databasePath: String,
```

Comment:
Would it be worth it to add tests to ensure we send the notifs from the RustPlaces and SQLiteReadingList?

Reply:
I'm working on tests right now, I was gonna add them as a separate PR but maybe I should get into the habit of making them part of the PR they affect so I don't run the risk of getting distracted and skipping them 🤔